### PR TITLE
FEAT: add xUnit logging implementation

### LINKED ISSUE #3: Configure CI build
Configure a CI build that will build & test the changes made in every PR

### LINKED ISSUE #8: Provide a release pipeline for MyGet.org
Provide a release pipeline for MyGet.org.

### Checklist

- [ ] Configure to run for every PR

- [ ] Use 'Determine NuGet Package Version' task group to determin package version

- [ ] Build the codebase

- [ ] Run test suite

- [ ] Push all NuGet packages to MyGet.org

### LINKED ISSUE #9: Provide a release pipeline for NuGet.org
Provide a release pipeline for NuGet.org.

### Checklist

- [ ] Build the codebase

- [ ] Run test suite

- [ ] Tag the codebase on success

- [ ] Create a GitHub pre-release for a preview releases

- [ ] Create a GitHub release for full releases

- [ ] Push all NuGet packages to NuGet.org

### PR DESCRIPTION
**Pull request by stijnmoreels**
Provides a `XunitTestLogger` in a new `Arcus.Testing.Logging` library.

Relates to https://github.com/arcus-azure/arcus/issues/19
Closes #3 
Closes #8
Closes #9

### CODE REVIEW

File: src/Arcus.Template/XunitTestLogger.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using GuardNet;
+using Microsoft.Extensions.Logging;
+using Xunit.Abstractions;
+
+namespace Arcus.Testing.Logging
```
Comment (tomkerkhove) [inline src/Arcus.Template/XunitTestLogger.cs]: Shouldn't this file be in `src/Arcus.Template.Logging/XunitTestLogger.cs`?

Comment (stijnmoreels) [reply src/Arcus.Template/XunitTestLogger.cs]: Ah, yes, have forgotton to change to folder structure.

**Comment by stijnmoreels** [comment]
I'm not sure what you mean by the "setup tool"? I've created the repo based on the arcus template repo.

**Comment by stijnmoreels** [comment]
TODO: basic intro docs and simple feature docs for logging.

**Comment by tomkerkhove** [comment]
> I'm not sure what you mean by the "setup tool"? I've created the repo based on the arcus template repo.

https://github.com/arcus-azure/arcus#starting-a-new-components

**Comment by stijnmoreels** [comment]
> > I'm not sure what you mean by the "setup tool"? I've created the repo based on the arcus template repo.
> 
> https://github.com/arcus-azure/arcus#starting-a-new-components

Ok, sorry, ran the tool, already added the pipelines.

**Pull request merged**